keyboards/handwired/atreus50/keymaps/ajp10304/rules.mk

```diff
@@ -6,3 +6,4 @@ OPT_DEFS = $(filter-out -DBOOTLOADER_SIZE=4096,$(TEMP))
 OPT_DEFS += -DBOOTLOADER_SIZE=512
```

Comment:
```suggestion
```
You shouldn't need this or the previous two lines, since you've set `BOOTLOADER = halfkay`.